src/bluesearch/database/article.py

```diff
@@ -15,29 +15,17 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 """Abstraction of scientific article data and related tools."""
+from __future__ import annotations
+
+import html
 from abc import ABC, abstractmethod
 from dataclasses import dataclass
 from pathlib import Path
-from typing import (
-    Dict,
-    Generator,
-    Iterable,
-    List,
-    Optional,
-    Sequence,
-    Tuple,
-    Type,
-    TypeVar,
-    Union,
-)
+from typing import Generator, Iterable, Sequence
 from xml.etree.ElementTree import Element  # nosec
 
 from defusedxml import ElementTree
 
-# This is for annotating the return value of the Article.parse class method, see
-# https://github.com/python/typing/issues/254#issuecomment-661803922
-_T = TypeVar("_T", bound="Article")
```

Comment:
good riddance! Great to see that newer Pythons don't need this construct

Reply:
He he. It might still  be necessary if we want to annotate the type of `cls`. But I just removed it and `mypy` seems to be happy with `cls` not being annotated, even with `disallow_incomplete_defs = true`. This setting is not enabled in our config at the moment, but maybe we should consider adding it.